### ISSUE TITLE
Unable to extract BuildManifest

### ISSUE THREAD

**Issue by roeegh** [airsquared/blobsaver#448]
**Describe the bug**
Unable to save blobs

**Screenshots**
<img width="385" alt="image" src="https://user-images.githubusercontent.com/62909766/173670261-a061c9d5-0109-4ff8-83b7-c646c871d392.png">

**Desktop:**
 - OS Version: MacBook Air M1 (Ventura 13.0)
 - iOS Device: iPhone 13 Pro Max (iOS 15.1.1)

**Checklist (place an x between the brackets to mark as completed):**
 - [x] I have made sure I am on the latest version of blobsaver
 - [x] I have checked other open issues to see if my problem has already been reported
 - [x] I have tried common troubleshooting tactics such as restarting my computer and reinstalling blobsaver

**Log(s)**
```
airsquared.blobsaver.app.TSS$TSSException: Unable to extract BuildManifest.
	at airsquared.blobsaver@3.2.0/airsquared.blobsaver.app.TSS.saveFor(Unknown Source)
	at airsquared.blobsaver@3.2.0/airsquared.blobsaver.app.TSS.call(Unknown Source)
	at airsquared.blobsaver@3.2.0/airsquared.blobsaver.app.TSS.call(Unknown Source)
	at javafx.graphics@18/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: Error on ZipFile ipsw
	at blobsaver.merged.module@3.2.0/org.apache.commons.compress.archivers.zip.ZipFile.<init>(Unknown Source)
	at blobsaver.merged.module@3.2.0/org.apache.commons.compress.archivers.zip.ZipFile.<init>(Unknown Source)
	at airsquared.blobsaver@3.2.0/airsquared.blobsaver.app.Utils.extractManifestFromZip(Unknown Source)
	at airsquared.blobsaver@3.2.0/airsquared.blobsaver.app.Utils.extractBuildManifest(Unknown Source)
	... 8 more
	Suppressed: java.lang.NullPointerException: Cannot invoke "java.nio.channels.ReadableByteChannel.close()" because "this.ch" is null
		at airsquared.blobsaver@3.2.0/airsquared.blobsaver.app.Network$HttpChannel.close(Unknown Source)
		... 10 more
Caused by: javax.net.ssl.SSLHandshakeException: TLS Server certificate issued after 2019-12-31 and anchored by a distrusted legacy Symantec root CA: CN=GeoTrust Primary Certification Authority - G2, OU=(c) 2007 GeoTrust Inc. - For authorized use only, O=GeoTrust Inc., C=US
	at java.base/sun.security.ssl.Alert.createSSLException(Unknown Source)
	at java.base/sun.security.ssl.TransportContext.fatal(Unknown Source)
	at java.base/sun.security.ssl.TransportContext.fatal(Unknown Source)
	at java.base/sun.security.ssl.TransportContext.fatal(Unknown Source)
	at java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.checkServerCerts(Unknown Source)
	at java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.onConsumeCertificate(Unknown Source)
	at java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.consume(Unknown Source)
	at java.base/sun.security.ssl.SSLHandshake.consume(Unknown Source)
	at java.base/sun.security.ssl.HandshakeContext.dispatch(Unknown Source)
	at java.base/sun.security.ssl.HandshakeContext.dispatch(Unknown Source)
	at java.base/sun.security.ssl.TransportContext.dispatch(Unknown Source)
	at java.base/sun.security.ssl.SSLTransport.decode(Unknown Source)
	at java.base/sun.security.ssl.SSLSocketImpl.decode(Unknown Source)
	at java.base/sun.security.ssl.SSLSocketImpl.readHandshakeRecord(Unknown Source)
	at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source)
	at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source)
	at java.base/sun.net.www.protocol.https.HttpsClient.afterConnect(Unknown Source)
	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(Unknown Source)
	at airsquared.blobsaver@3.2.0/airsquared.blobsaver.app.Network$HttpChannel.ensureOpen(Unknown Source)
	at airsquared.blobsaver@3.2.0/airsquared.blobsaver.app.Network$HttpChannel.size(Unknown Source)
	at blobsaver.merged.module@3.2.0/org.apache.commons.compress.archivers.zip.ZipFile.tryToLocateSignature(Unknown Source)
	at blobsaver.merged.module@3.2.0/org.apache.commons.compress.archivers.zip.ZipFile.positionAtEndOfCentralDirectoryRecord(Unknown Source)
	at blobsaver.merged.module@3.2.0/org.apache.commons.compress.archivers.zip.ZipFile.positionAtCentralDirectory(Unknown Source)
	at blobsaver.merged.module@3.2.0/org.apache.commons.compress.archivers.zip.ZipFile.populateFromCentralDirectory(Unknown Source)
	... 12 more
Caused by: sun.security.validator.ValidatorException: TLS Server certificate issued after 2019-12-31 and anchored by a distrusted legacy Symantec root CA: CN=GeoTrust Primary Certification Authority - G2, OU=(c) 2007 GeoTrust Inc. - For authorized use only, O=GeoTrust Inc., C=US
	at java.base/sun.security.validator.SymantecTLSPolicy.checkNotBefore(Unknown Source)
	at java.base/sun.security.validator.SymantecTLSPolicy.checkDistrust(Unknown Source)
	at java.base/sun.security.validator.CADistrustPolicy$1.checkDistrust(Unknown Source)
	at java.base/sun.security.validator.EndEntityChecker.check(Unknown Source)
	at java.base/sun.security.validator.Validator.validate(Unknown Source)
	at java.base/sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source)
	at java.base/sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source)
	... 35 more
```

**Comment by rlaphoenix** [comment]
Happens to me too on Windows 11.

**Comment by vraixen** [comment]
me too on windows 11

**Comment by airsquared** [comment]
Duplicate of #445; This has been fixed in v3.2.1.
